dev/bots/prepare_package.dart

```diff
@@ -370,6 +370,12 @@ class ArchiveCreator {
     await _runGit(<String>['clean', '-f', '-X', '**/.packages']);
     /// Remove package_config files and any contents in .dart_tool
     await _runGit(<String>['clean', '-f', '-X', '**/.dart_tool']);
+    /// Remove git subfolder from .pub-cache, this contains the flutter goldens
+    /// and new flutter_gallery.
+    final Directory gitCache = Directory(path.join(flutterRoot.absolute.path, '.pub-cache', 'git'));
```

Comment:
Rather than excluding by git, could we just exclude everything from the `dev/` folder specifically?

That way, if we move to distributing the gallery some other way (e.g. by pub somehow?) this won't break.